vividus-extension-visual-testing/src/main/resources/spring.xml

```diff
@@ -7,7 +7,7 @@
        default-lazy-init="true">
 
     <beans profile="web">
-        <bean id="coordsProvider" class="org.vividus.visual.screenshot.AdjustingCoordsProvider" />
+        <bean id="coordsProvider" class="org.vividus.visual.screenshot.WebAdjustingCoordsProvider" />
     </beans>
```

Comment:
to move to `vividus-plugin-web-app`?

Reply:
I would not to, because now it is used only inside visual testing module